source/chapters/appendix.rst

```diff
@@ -284,22 +284,59 @@ Debugger Enabled                          :kbd:`Ctrl` + :kbd:`Shift` + :kbd:`D`
 
 .. _appendix-command-line-options:
 
+Launching Mixxx from the Command Line
+=====================================
+To launch Mixxx using the command line:
+
+**Windows**
+  Locate the folder where Mixxx is installed and copy the file path. You can also obtain the file path by right clicking on the Mixxx icon and then choosing 'Properties'.
+  Copy the path and paste it in terminal and add ``\mixxx`` at the end - everything should be inside quotes. Press Enter. The entire command will look like this ::
+
+    "C:\Program Files\Mixxx\mixxx"
+
+  You could also choose to first ::
+
+    cd "C:\Program Files\Mixxx"
+
+  and then enter ::
+
+    mixxx
+
+
+**Mac OSX**
```

Comment:
Apple now stylizes it as `macOS` since 2016: https://en.wikipedia.org/wiki/MacOS_Sierra